src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuName.cs

```diff
@@ -14,6 +14,9 @@ namespace Microsoft.Azure.Management.Analysis.Models
     /// </summary>
     public static class SkuName
     {
+        public const string B1 = "B1";
+        public const string B2 = "B2";
```

Comment:
@igor-uzhviev you will need to bump up the version in .csproj and AssemblyInfo.cs